### ISSUE TITLE
LDAP issues 

### ISSUE THREAD

**Issue by osnet** [Programmierus/ldap-mailcow#6]
Hi, 
great tool btw ... 

remember to add  this in the docker-compose.yml 
 - ./data/conf/dovecot/extra.conf:/conf/dovecot/extra.conf:ro
this prevents overwrite if there are any public folders set.

the file  ./data/conf/sogo/plist_ldap  gets written with  ldap://ldap//hostname:port

with your addon one is now able to login with  email-address  AND  sAMAaccountName    GREAT

But there is still an issue loggin in via EAS / Active Sync ...  you still geht 401 if you dont provide your local password ...

**Comment by Programmierus** [comment]
Hi, sorry I have not much time now to finish some work on this tool... 

As about to config redirection - the tool makes backups for all configs it overwrites...

EAS/ActiveSync/Web GUI please take a look here #5 - I have proposed a solution...

**Comment by Programmierus** [comment]
So, I've added some changes today (507dd95).

I can reconfirm there is **no need** to specially restrict extra.conf to read-only: ldap-mailcow backups all scripts.

I have added information to README about enabling WebUI & EAS.

**Comment by etique57** [comment]
Hello @Programmierus 

Quick question, the extra.conf is indeed backed up but it keeps being rewritten. I already have a command there (disabling encryption for Dovecot) and I lose it each time I start the container. I remember I saw a workaround somewhere but can't find it anymore.

Thanks for any hint!

Ok I found the solution: I put my modification in the template/dovecot/extra.conf file directly :)
